app/classes/Transvision/ShowResults.php

```diff
@@ -58,6 +58,9 @@ public static function getSuggestionsResults($source_strings, $target_strings, $
         ];
         $flat_output = [];
 
+        // Make sure $max_results is never set to 0, in which case we set it to 10
```

Comment:
@flodolo I don't have any idea about what i should be doing for the functional test, it was Pascal request so i did it.